File: core-java-practice/gcr-codebase/pragramming-elements/areaOfCircle.java
import java.util.Scanner;

public class areaOfCircle {
    public static void main(String[] args) {
        //  Scanner for read input from the user
        Scanner sc = new Scanner(System.in);
        //  input radius of a circle
        int radius = sc.nextInt();
        double area = Math.PI*radius*radius;
        System.out.println("Area of Circle: "+ area);
        sc.close();
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/AreaOfCircle.cs
using System;

class AreaOfCircle{
	
	static void Main()
	{
		Console.Write("Enter Radius of Circle: ");
		int radius = int.Parse(Console.ReadLine());
		
		double area = 3.14 * radius * radius;
		
		Console.WriteLine($"Area of Circle: {area}");
	}
}